AccountingSystem/private/DataCache.py

```diff
@@ -43,7 +43,7 @@ def getReportData( self, reportRequest, reportHash, dataFunc ):
     Get report data from cache if exists, else generate it
     """
     reportData = self.__dataCache.get( reportHash )
-    if reportData == False:
+    if reportData is None:
```

Comment:
Nope, I cant remind you, as I don't remember either. It was in a pull request too, for sure though.